build.gradle

```diff
@@ -206,6 +206,7 @@ pmd {
   toolVersion = "6.41.0"
   sourceSets = [sourceSets.main, sourceSets.test, sourceSets.functionalTest, sourceSets.integrationTest]
   reportsDir = file("$project.buildDir/reports/pmd")
+  ignoreFailures = true
```

Comment:
if we're effectivey ignoring PMD - should we remove the plugin etc ?

Reply:
It has been agreed that we ignore it for the moment there is a separated ticket that cameron is working to cover that.